src/signals/incident-management/containers/Dashboard/styled.ts

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2023 Gemeente Amsterdam
-import { Row, themeColor, themeSpacing } from '@amsterdam/asc-ui'
+import { themeColor, themeSpacing } from '@amsterdam/asc-ui'
 import styled from 'styled-components'
 
-export const StyledRow = styled(Row)`
+export const StyledRow = styled.div`
```

Comment:
using row would mean a padding for all the underlying components. Not really needed in some parts of the children.